services/common/src/components/reports/AddReportDetails.tsx

```diff
@@ -0,0 +1,476 @@
+                disabledDate={(currentDate) => currentDate.date() > moment().date}
+              />
+            </Form.Item>
+          </Form>
+        </Col>
+
+        <Col span={12}>
+          <Form layout="vertical">
+            <Form.Item label="Submitter Name">
+              <Input placeholder="Enter Name" style={{ borderWidth: "1px" }} />
+            </Form.Item>
+          </Form>
+        </Col>
+        <Col span={12}>
+          <Form layout="vertical">
+            <Form.Item label="Submitter Email">
+              <Input placeholder="Enter Email" style={{ borderWidth: "1px" }} />
+            </Form.Item>
+          </Form>
+        </Col>
+        <Col span={24}>
+          <Form layout="vertical">
+            <Form.Item label="Report Title and Additional Comment">
+              <Input.TextArea
+                rows={3}
+                placeholder="Include a concise and descriptive title to the report"
+                style={{ borderWidth: "1px" }}
+              />
+              Include a concise and descriptive title to the report
+            </Form.Item>
+          </Form>
+        </Col>
+
+        <Col span={24}>
+          <Form layout="vertical">
+            <Alert
+              style={{ borderWidth: "1px" }}
+              message=""
+              description={
+                <div>
+                  <Row>
+                    <Typography.Title level={5}>
+                      Your mine manager will be notified of this submission
+                    </Typography.Title>
+                  </Row>
+                  <Row>
+                    <Col span={12}>
+                      <Typography.Text>Mine Manager</Typography.Text>
+                      <Typography.Paragraph>Denise George</Typography.Paragraph>
+                    </Col>
+                    <Col span={12}>
+                      <Typography.Text>Mine Manager Email</Typography.Text>
+                      <Typography.Paragraph>Denise.George@gov.bc.ca</Typography.Paragraph>
+                    </Col>
+                    <Col span={24}>
+                      <Typography.Text>
+                        If this information is incorrect, please contact us.
+                      </Typography.Text>
+                    </Col>
+                  </Row>
+                </div>
+              }
+              type="info"
+            />
+          </Form>
+        </Col>
+
+        <Col span={24}>
+          <Typography.Title className="margin-large--top" level={4}>
+            Report File(s)
+          </Typography.Title>
+          <Form layout="vertical">
+            <ReportSubmissions
+              mineGuid={mineGuid}
+              mineReportSubmissions={props.mineReportSubmissions}
+              updateMineReportSubmissions={updateMineReportSubmissions}
+            />
+          </Form>
+        </Col>
+
+        <Col span={24}>
+          <Form layout="vertical">
+            <ReportsTable
```

Comment:
🤔 I think this might not be the correct table.  I think the designs are just asking for a table of the documents, not the other reports.

I see you brought over the editReport modal and a few other things that are related to this table, so it would probably mean that we can leave those back where they were.

<img width="585" alt="image" src="https://github.com/bcgov/mds/assets/83598933/ccc27643-bfe7-4bee-8ab9-d807925bb3d4">

Reply:
As discussed added a new component.